app/models/where_did_you_hear.rb

```diff
@@ -2,7 +2,8 @@ class WhereDidYouHear < ActiveRecord::Base
   before_validation :lookup_mappings
 
   def lookup_mappings
-    self.heard_from       = CodeLookup.for(value: heard_from_code)
-    self.pension_provider = CodeLookup.for(value: pension_provider_code)
+    %w(heard_from pension_provider).each do |field|
```

Comment:
I don't think this is any clearer / more concise than the code it replaced, to be fair.